docs/source/nse_table.ipynb

```diff
@@ -0,0 +1,250 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "1efd221d-8e96-4001-aa92-0b322f9eec79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nucs = [Nucleus(\"p\"), Nucleus(\"n\"), Nucleus(\"he4\"),\n",
+    "        Nucleus(\"fe52\"), Nucleus(\"fe53\"), Nucleus(\"fe54\"),\n",
+    "        Nucleus(\"fe55\"), Nucleus(\"fe56\"),\n",
+    "        Nucleus(\"co54\"), Nucleus(\"co55\"), Nucleus(\"co56\"),\n",
+    "        Nucleus(\"ni56\"), Nucleus(\"ni57\")]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "1f718159-5e68-40d6-ae8f-3f43c16a3da7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "warning: He4 was not able to be linked\n",
+      "warning: Fe53 was not able to be linked\n",
+      "warning: Fe52 was not able to be linked\n",
+      "warning: Ni57 was not able to be linked\n"
+     ]
+    }
+   ],
+   "source": [
+    "tl = pyna.TabularLibrary().linking_nuclei(nucs)\n",
+    "rl = pyna.ReacLibLibrary().linking_nuclei(nucs)\n",
+    "all_lib = tl + rl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "a46d080f-fa3d-433d-8e3b-a91ef98996d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dupes = all_lib.find_duplicate_links()\n",
+    "rates_to_remove = []\n",
+    "for d in dupes:\n",
+    "    rates_to_remove += [r for r in d if isinstance(r, pyna.rates.ReacLibRate)]\n",
+    "\n",
+    "for r in rates_to_remove:\n",
+    "    all_lib.remove_rate(r)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "3f2cba9e-ad5d-487e-9e45-4c896fa43559",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nse = pyna.NSENetwork(libraries=[all_lib])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8368c1a-53d9-4f33-ba7c-d40d2963610f",
+   "metadata": {},
+   "source": [
+    "Now we'll create a the grid of temperature, density, and $Y_e$ where we want to compute the NSE state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "d2a04dbe-a62b-4f90-bef9-4f6bde881509",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "223f5be5-32b8-429f-ae86-79ace7123d05",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Ts = np.logspace(9.6, 10.4, 3)\n",
+    "rhos = np.logspace(7, 10, 4)\n",
+    "yes = np.linspace(0.43, 0.5, 3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7172990e-4eca-4b9c-b1a4-f07ff2cc0f42",
+   "metadata": {},
+   "source": [
+    "Finally we can generate the table.  This will compute the NSE state at each combination of $(\\rho, T, Y_e)$.  To help accelerate the convergence,\n",
+    "it start at the highest temperature and loop over $\\rho$ and $Y_e$ and cache the values of the proton and neutron chemical potentials for the next\n",
+    "temperature."
```

Comment:
```suggestion
    "it will start at the highest temperature and loop over $\\rho$ and $Y_e$ and cache the values of the proton and neutron chemical potentials for the\n",
    "next temperature."
```